packages/cli/src/__tests__/fixtures/studio-test-project-schema-folder/schema/A.prisma

```diff
@@ -1,10 +1,9 @@
 datasource my_db {
   provider = "sqlite"
-  url      = "file:./dev_tmp.db"
+  url      = "file:../dev_tmp.db"
```

Comment:
ℹ️  Actually the test was wrong so far. Aka expected the buggy behavior 😅 Although the db file "should" have been in the directory next to the `A.prisma` file according to this path, it was actually placed a level higher. ;)